clients/js/package.json

```diff
@@ -35,35 +35,30 @@
     "format": "prettier --cache --write . && eslint --ignore-path .gitignore --fix .",
     "build": "npm run build:esm && npm run build:cjs",
     "clean": "rm -rf lib",
-    "build:esm": "tsc -p ./tsconfig.json",
+    "build:esm": "tsc -p ./tsconfig.esm.json",
     "build:cjs": "tsc -p ./tsconfig.cjs.json && node scripts/rename-cjs",
-    "test": "jest",
-    "coverage": "jest --coverage",
+    "test": "jest --coverage",
     "prepublishOnly": "npm run build",
-    "proxy": "node --loader ts-node/esm  scripts/proxy.ts"
+    "proxy": "node --loader ts-node/esm scripts/proxy.ts"
   },
   "dependencies": {
-    "@oasisprotocol/deoxysii": "0.0.5",
-    "cborg": "1.10.2",
-    "ethers": "6.10.0",
     "@noble/hashes": "1.3.2",
-    "tweetnacl": "1.0.3",
-    "type-fest": "2.19.0"
+    "@oasisprotocol/deoxysii": "0.0.5",
+    "cborg": "1.10.2"
   },
   "devDependencies": {
     "@types/jest": "^29.5.11",
     "@types/node": "^18.7.18",
-    "@types/node-fetch": "^2.6.2",
     "@typescript-eslint/eslint-plugin": "^5.38.0",
     "@typescript-eslint/parser": "^5.38.0",
     "eslint": "^8.23.1",
     "eslint-config-prettier": "^8.5.0",
+    "ethers": "6.x",
     "jest": "^29.7.0",
-    "nock": "^13.4.0",
-    "node-fetch": "^2.6.7",
     "prettier": "^2.7.1",
     "ts-jest": "^29.1.1",
     "ts-node": "^10.9.2",
+    "tweetnacl": "^1.0.3",
```

Comment:
This is only used in `cipher.spec.ts` and `utils.ts`, this could be switched to `@noble/curves`.

The bug in question:
> IMPORTANT BUG FIX. Due to a bug in calculating carry in
modulo reduction that used bit operations on integers larger than
32 bits, nacl.sign or nacl.sign.detached could have created
incorrect signatures.
> 
> This only affects signing, not verification.